cypress/e2e/group2/myworkflows.ts

```diff
@@ -113,6 +116,32 @@ describe('Dockstore my workflows', () => {
       // cy.contains('2020-02-19T18:20');
       // cy.contains('2020-06-05T07:40');
       cy.contains('1 – 2 of 2');
+
+      //Filtering
+      const filteredResponse = [
```

Comment:
Missed this in my first pass, but you should explicitly declare the type here: `const filteredResponse: LambdaEvent[] = [`.

That way, if we ever change the Lambda event schema in the web service, this code will fail to compile, which will be good, because it will force us to keep the mock in sync with the web service schema.